src/config/index.ts

```diff
@@ -16,7 +16,7 @@ const debug = env !== 'production'
 dotenv.config({ path: configFile, debug: debug })
 
 // retrieve port from env or use default
-const port = process.env.PORT || env === 'production' ? 5000 : 5002
+const port = process.env.PORT || (env === 'production' ? 5000 : 5002)
```

Comment:
It was taking `(process.env.PORT || env === 'production') ? 5000 : 5002` before

Reply:
Good catch! 👍

Reply:
👍